docs/using-the-jdbc-driver/Telemetry.md

```diff
@@ -44,7 +44,30 @@ Before enabling the Telemetry feature, a few setup steps are required to ensure
 
 Telemetry for the AWS JDBC Driver is a monitoring strategy that overlooks all plugins enabled in [`wrapperPlugins`](../UsingTheJdbcDriver.md#connection-plugin-manager-parameters) and is not a plugin in itself. Therefore no changes are required in the `wrapperPlugins` parameter to enable Telemetry.
 
-In order to enable Telemetry in the AWS JDBC Driver all you need to do is to set the `enableTelemetry` property to `true` either in the code or in the connection string.
+In order to enable Telemetry in the AWS JDBC Driver you need to first set the `enableTelemetry` property to `true` either in the code or in the connection string.
+
+After that, it will be needed to, in the application code that will be using the driver, to set up the recorders that will take care of exporting the telemetry data from the code to the ADOT Collector.
+
+Below you find an example on how to set up the OpenTelemetry SDK and its configurations:
```

Comment:
Can a link to the example file be added here? And on the main examples table on the readme: 
 https://github.com/awslabs/aws-advanced-jdbc-wrapper#examples